octo/test_test.go

```diff
@@ -0,0 +1,107 @@
+package octo_test
```

Comment:
You are the best for including these tests!! It's sad this repo never had any. We do need to keep consistent though. Can you use `github.com/sclevine/spec` and Gomega?